emissions.md

```diff
@@ -1,29 +1,41 @@
 ---
+title: "Emissions"
 layout: index
-title: GCAM Emissions
-prev: hector.html
-next: choice.html
+prev: supply_energy.html
+next: outputs_emissions.html
 gcam-version: v5.3
 ---
-
 ## Overview
 
 GCAM projects emissions of a suite of greenhouse gases (GHGs) and air pollutants:
 
 CO<sub>2</sub>, CH<sub>4</sub>, N<sub>2</sub>O, CF<sub>4</sub>, C<sub>2</sub>F<sub>6</sub>, SF<sub>6</sub>, HFC23, HFC32, HFC43-10mee, HFC125, HFC134a, HFC143a, HFC152a, HFC227ea, HFC236fa, HFC245fa, HFC365mfc, SO<sub>2</sub>, BC, OC, CO, VOCs, NO<sub>x</sub>, NH<sub>3</sub>
 
-Future emissions are determined by the evolution of drivers (such as energy consumption, land-use, and population) and technology mix. How this is represented in GCAM varies by emission type. 
+Future emissions are determined by the evolution of drivers (such as energy consumption, land-use, and population) and technology mix. How this is represented in GCAM varies by emission type.
```

Comment:
Future emissions are determined by the evolution of drivers (such as energy consumption, land-use, and population), technology mix, and abatement measures